scripts/submit_skims2017_mib.sh

```diff
@@ -17,40 +17,34 @@ source /cvmfs/cms.cern.ch/cmsset_default.sh
 source scripts/setup.sh
 mkdir $OUTDIRR
 
-<<COMMENT1
-
 ###################
 # TT
-# TT x section: 831.76 for inclusive sample, W->had 67,60% , W->l nu 3*10,8% = 32,4% (sum over all leptons)
-# hh = 45.7%
-# ll = 10.5%
-# hl = 21.9% (x2 for permutation t-tbar)
-# technical note: use -q long if splitting on ~180-350 jobs
-
```

Comment:
since I don't see tany changes in the XSs for TT, please put back this explanation as I think it can always be useful!

Reply:
Done!